packages/spright-components/src/chat/types.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * A message type in a chat conversation.
+ * @public
+ */
+export const ChatMessageType = {
```

Comment:
We have a pattern for writing tests for our types, since the syntax is kind of special. You can copy that pattern from any other `types.spec.ts` file in the repo.